book/linearalgebra.tex

```diff
@@ -8901,5 +8908,77 @@ \section*{Eigenvectors}
 
 \end{bookonly}
 
+\newpage
+
+\begin{tutorial}
+	\Title{Math \& the World}
+	
+	\Heading{Learning Objectives}
+	\begin{objectives}
+			In this tutorial you will work on rephrasing problems
+			with mathematical language---this is an essential skill if you ever
+			plan on applying mathematical techniques to the world!
+
+			These problems relate to the following course learning
+			objective: \textit{work independently to understand concepts and procedures that have not
+			been previously explained to you}.
+	\end{objectives}
```

Comment:
Keep the learning objectives in the tutorial file, and leave things as `\section*`, etc.. You can keep `\Title` in this file. I'd like to have it so that the same sources are included for both the book and the stand-alone tutorials.